concourse/pipelines/windows-image-build.jsonnet

```diff
@@ -7,6 +7,7 @@ local gcp_secret_manager = import '../templates/gcp-secret-manager.libsonnet';
 local client_envs = ['testing', 'staging', 'internal'];
 local server_envs = ['testing', 'staging', 'internal', 'prod'];
 local sql_envs = ['testing', 'staging', 'prod'];
+local windows_install_media_envs = ['testing', 'staging', 'prod'];
```

Comment:
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/build-publish/windows/windows-install-media.publish.json

Only includes testing and prod.

Reply:
done